includes/admin/home/tasks/task/class-sensei-home-task-configure-learning-mode.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * File containing the Sensei_Home_Task_Configure_Learning_Mode class.
+ *
+ * @package sensei-lms
+ * @since $$next-version$$
+ */
+
+/**
+ * Sensei_Home_Task_Configure_Learning_Mode class.
+ *
+ * @since $$next-version$$
+ */
+class Sensei_Home_Task_Configure_Learning_Mode implements Sensei_Home_Task {
+
+	/**
+	 * The ID for the task.
+	 *
+	 * @return string
+	 */
+	public function get_id(): string {
+		return 'configure-learning-mode';
+	}
+
+	/**
+	 * Number used to sort in frontend.
+	 *
+	 * @return int
+	 */
+	public function get_priority(): int {
+		return 200;
+	}
+
+	/**
+	 * Task title.
+	 *
+	 * @return string
+	 */
+	public function get_title(): string {
+		return __( 'Configure Learning Mode', 'sensei-lms' );
+	}
+
+	/**
+	 * Task url.
+	 *
+	 * @return string
+	 */
+	public function get_url(): ?string {
+		return admin_url( 'edit.php?post_type=course&page=sensei-settings#appearance-settings' );
+	}
+
+	/**
+	 * Task image.
+	 *
+	 * @return string|null
+	 */
+	public function get_image(): ?string {
+		// TODO Add image path.
+		return null;
+	}
+
+	/**
+	 * Whether the task is completed or not.
+	 *
+	 * @return bool
+	 */
+	public function is_completed(): bool {
+		$visited_settings_sections = get_site_option( Sensei_Settings::VISITED_SECTIONS_OPTION_KEY, [] );
```

Comment:
Couldn't we use `get_option` here?

I'm asking because, in a multisite installation, we probably don't want to mark the task for every website as done as soon as the user completes the task.

In the same way, in that scenario, given that we're NOT using [`update_site_option`](https://developer.wordpress.org/reference/functions/update_site_option/), this may not even work as expected.

[More details here](https://developer.wordpress.org/reference/functions/get_site_option/#more-information).